frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -121,11 +121,12 @@ export class NoticeOfWorkTable extends Component {
   });
 
   columns = () => [
-    {
-      title: "Source",
-      dataIndex: "source",
-      render: (text) => <div title="Source">{text}</div>,
-    },
+    // add to UI once we connect the backend
```

Comment:
This is a great spot to add `TODO:` . A lot of IDEs support organizing and navigating by TODO and FIXME inline comments.

https://www.jetbrains.com/help/idea/using-todo.html